src/screens/ProfileScreen/DetailScreen/ExtensionRequest.tsx

```diff
@@ -144,8 +143,11 @@ const styles = StyleSheet.create({
   },
   input: {
     borderColor: 'skyBlue',
-    borderwidth: 2,
+    borderWidth: 2,
     fontSize: 15,
     backgroundColor: 'lightgrey',
   },
+  whiteText: {
+    color: 'white',
```

Comment:
use color variable

Reply:
for color variable, we need to add a new issue  as we have to update it everywhere with testing.